lib/locales/es.ts

```diff
@@ -187,55 +187,67 @@ export const locale: Locale = {
+    apollinaris_of_ravenna_bishop: 'San Apolinar de Rávena, Obispo y Mártir',
     arbogast_of_strasbourg_bishop: 'San Arbogast, Obispo',
-    arbogast_of_strasbourg_bishop_patron_of_the_archdiocese_of_strasbourg:
-      'San Arbogast, Obispo y Patrono de la Arquidiócesis de Estrasburgo',
+    arbogast_of_strasbourg_bishop_patron_of_the_archdiocese_of_strasbourg: 'San Arbogast, Obispo y Patrono de la Arquidiócesis de Estrasburgo',
     ascension_of_the_lord: 'Ascensión del Señor',
     ash_wednesday: 'Miércoles de Ceniza',
     assumption_of_the_blessed_virgin_mary: 'La Asunción de la Santísima Vírgen María',
+    athanasius_of_alexandria_bishop: 'San Atanasio de Alejandría, Obispo y Doctor de la Iglesia',
     audoen_of_rouen_bishop: 'San Audoeno, Obispo',
     augustine_of_canterbury_bishop: 'San Agustín de Canterbury, Obispo',
     augustine_of_hippo_bishop: 'San Agustín de Hipona, Obispo y Doctor de la Iglesia',
+    augustine_zhao_rong_priest_and_companions_martyrs: 'San Agustín Zhao Rong, Sacerdote, y compañeros, Mártires',
     aurelia_of_strasbourg_virgin: 'Santa Aurelia de Estrasburgo, Vírgen',
     baptism_of_the_lord: 'Bautismo del Señor',
     barbara_of_heliopolis_virgin: 'Santa Bárbara, Vírgen y Mártir',
     barnabas_apostle: 'San Bernabé, Apóstol',
     bartholomew_apostle: 'San Bartolomeo, Apóstol',
+    bartholomew_dias_laurel_religious: 'Beato Bartolomé Días Laurel, Religioso y Mártir',
+    basil_the_great_and_gregory_nazianzen_bishops: 'San Basilio el Grande y San Gregorio Nacianceno, Obispos y Doctores de la Iglesia',
     beatrice_da_silva_meneses_virgin: 'Santa Beatriz da Silva, Vírgen',
     bede_the_venerable_priest: 'San Beda el Venerable, Sacerdote y Doctor de la Iglesia',
     benedict_of_aniane_abbot: 'San Benito de Aniano, Abad',
     benedict_of_jesus_valdivielso_saez_religious: 'San Héctor Valdivielso Sáez, Mártir',
+    benedict_of_nursia_abbot: 'San Benito de Nursia, Abad',
     bernard_of_clairvaux_abbot: 'San Bernardo de Clairvaux, Abad y Doctor de la Iglesia',
     bernardine_of_siena_priest: 'San Bernardino de Siena, Sacerdote',
     beuno_of_wales_abbot: 'San Beuno, Abad',
     blaise_of_sebaste_bishop: 'San Blás, Obispo y Mártir',
     blessed_martyrs_of_paris: 'Beatos Mártires de París en la Revolución Francesa',
     bonaventure_of_bagnoregio_bishop: 'San Buenaventura, Obispo y Doctor',
     boniface_of_mainz_bishop: 'San Bonifacio, Obispo y Mártir',
+    bridget_of_sweden_religious: 'Santa Brígida de Suecia',
     brigid_of_kildare_virgin: 'Santa Brígida de Kildare, Abadesa',
     brigid_of_kildare_virgin_copatroness_of_ireland: 'Santa Brígida de Kildare, Vírgen y Patrona Secundaria de Irlanda',
     bruno_of_cologne_priest: 'San Bruno, Sacerdote',
+    cajetan_of_thiene_priest: 'San Cayetano de Thiene, Sacerdote',
+    callistus_i_pope: 'San Calixto I, Papa y Mártir',
+    camillus_de_lellis_priest: 'San Camilo de Lelis (Lellis), Sacerdote',
```

Comment:
Based on [this](https://liturgiapapal.org/attachments/article/1137/Calendario%2022-23.pdf#page=90) and [this](https://www.conferenciaepiscopal.es/wp-content/uploads/2022/11/CLP-y-salmos-responsoriales-2022-2023_internet.pdf#page=246) ordos, I suggest to change this.

Note that I haven’t checked all celebration names, that could be a task for you if you want.

```suggestion
    camillus_de_lellis_priest: 'San Camilo de Lelis, presbítero',
```